pontoon/administration/files.py

```diff
@@ -53,8 +66,8 @@ def split_comments_mine(
             cls.split_entities(
                 text, object, code=code, pointer=pointer, end=st0)
         groups = match.groups()
-        comment = silme.core.structure.Comment(
-            match.group(0)[1:].replace('\n#', '\n'))
+        comment = Comment()
+        comment.add(match.group(0)[1:].replace('\n#', '\n'))
```

Comment:
1 year? Jeebus. We should just fork the code to Github, apply our fix, and then pull in that version. All this replicated code is kinda scary to me.